lib/explorer/backend/lazy_series.ex

```diff
@@ -1177,19 +1194,28 @@ defmodule Explorer.Backend.LazySeries do
   end
 
   @impl true
-  def re_named_captures(_series, _pattern) do
-    raise """
-    #{unsupported(:re_named_captures, 2)}
+  def re_named_captures(series, pattern) do
+    lazy_s = lazy_series!(series)
 
-    If you want to capture named groups from a column, you must do so outside of a query.
-    For example, instead of:
+    target_dtype = get_backend(lazy_s).re_dtype(pattern)
 
-        Explorer.DataFrame.mutate(df, new_column: re_named_captures(column, ~S/(a|b)/))
+    data = new(:re_named_captures, [lazy_s, pattern], target_dtype)
 
-    You must write:
+    Backend.Series.new(data, target_dtype)
+  end
 
-        Explorer.DataFrame.put(df, :new_column, Explorer.Series.re_named_captures(column, ~S/(a|b)/))
-    """
+  defp get_backend(%__MODULE__{} = lazy_series) do
+    lazy_series.backend || warning_with_default_backend()
+  end
+
+  defp warning_with_default_backend do
+    backend = Explorer.Backend.get()
+
+    Logger.warning(
+      "cannot get backend from LazySeries. Using the default one: #{inspect(backend)}"
+    )
```

Comment:
When would this be the case?

Reply:
For this function I couldn't find a way to reproduce the exception. But if a series operation that accepts scalar values for all its arguments is called, it will call "from_list", and it will raise here. But this is unlikely to happen IMHO.

Reply:
This log*

Reply:
What if we raise instead then?

Reply:
Yeah, it makes sense! I will update it later today :)

Reply:
1ae1ddd